tests/nvidiagpu/gpu_suite_test.go

```diff
@@ -32,11 +32,9 @@ var _ = JustAfterEach(func() {
 	reporter.ReportIfFailed(
 		specReport, currentFile, tsparams.ReporterNamespacesToDump, tsparams.ReporterCRDsToDump, clients.SetScheme)
 
-	dumpDir := inittools.GeneralConfig.GetDumpTestReportLocation(currentFile)
-	if dumpDir != "" {
-		artifactDir := fmt.Sprintf("%s/gpu-must-gather", dumpDir)
-		if err := reporter.RunMustGather(artifactDir, 5*time.Minute); err != nil {
-			glog.Errorf("Error running RunMustGather, %v", err)
-		}
+	dumpDir := inittools.GeneralConfig.GetReportPath("gpu-must-gather")
+	artifactDir := fmt.Sprintf("%s/gpu-must-gather", dumpDir)
```

Comment:
This will write the output into `<reports_dir>/gpu-must-gather/gpu-must-gather`, because `dumpDir` already equals `<reports_dir>/gpu-must-gather`. So you don't need to append another `gpu-must-gather` here.